### PR TITLE
Fix bug in see-also

### PR DESCRIPTION
**Pull request by qinezh**
@chenkennt @superyyrrzz @vwxyzh @hellosnow @ansyral

### CODE REVIEW

File: src/docfx.website.themes/docs.html/partials/class.tmpl.partial [hunk 1]
```diff
@@ -286,7 +286,7 @@
 <div class="level">
   <section class="primary">
      {{#type.specName}}
-        {{{value}}}
+        <code class="lang-{{lang}}">{{{value}}}</code>
```
Comment (hellosnow) [inline src/docfx.website.themes/docs.html/partials/class.tmpl.partial]: {{{?

**Comment by hellosnow** [comment]
:shipit:

File: src/docfx.website.themes/docs.html/partials/class.tmpl.partial (hunk 1 above)
Comment (qinezh) [inline src/docfx.website.themes/docs.html/partials/class.tmpl.partial]: the same as L165

Comment (hellosnow) [inline src/docfx.website.themes/docs.html/partials/class.tmpl.partial]: how about update together?

---
In reply to: [67851987](https://github.com/dotnet/docfx/pull/409#discussion_r67851987) [](ancestors = 67851987)

**Comment by superyyrrzz** [comment]
I find a page with seealso: https://docs.microsoft.com/en-us/rights-management/sdk/2.1/api/win/environment%20properties#msipc_environment_properties
This PR will change the current UI. What about `<span class="lang-{{lang}}">{{{value}}}</span>`?

**Comment by qinezh** [comment]
@superyyrrzz it doesn't work after trying to use <span>

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**